sdk/keyvault/keyvault-common/.eslintrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "plugins": ["@azure/azure-sdk"],
+  "extends": ["plugin:@azure/azure-sdk/azure-sdk-base"]
+}
```

Comment:
You can now delete this file because `eslint` will automatically pick up the one in the `sdk` directory.

Reply:
ohh I didn't know that ! thank you!